### PR TITLE
Test topology aware enablement

### PR DESCRIPTION
**Pull request by itzikb-redhat**
Make sure it's only enabled when there are no Compute availability zones without having Volume AZs with the same names

### CODE REVIEW

File: test/extended/openstack/topology.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
+package openstack
+
+import (
+
+	//	"github.com/gophercloud/gophercloud/v2"
```
Comment (rlobillo) [inline test/extended/openstack/topology.go:5]: please remove

Comment (rlobillo) [reply test/extended/openstack/topology.go:5]: same for the rest of commented imports

File: test/extended/openstack/topology.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+		cfg, err := e2e.LoadConfig()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		dc, err = dynamic.NewForConfig(cfg)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		clientSet, err = e2e.LoadClientset()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		skipUnlessMachineAPIOperator(ctx, dc, clientSet.CoreV1().Namespaces())
+	})
+	g.It("enables topology aware scheduling when compute and volume AZs are identical", func(ctx g.SpecContext) {
```
Comment (rlobillo) [inline test/extended/openstack/topology.go:47]: Would you mind to add a reference to the bug that resolved this issue as a comment?

File: test/extended/openstack/topology.go [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+)
+
+var _ = g.Describe("[sig-installer][Suite:openshift/openstack] The OpenShift cluster", func() {
+	defer g.GinkgoRecover()
+
+	var dc dynamic.Interface
+	var clientSet *kubernetes.Clientset
+	oc := exutil.NewCLI("openstack")
+
+	//var computeClient *gophercloud.ServiceClient
```
Comment (MaysaMacedo) [inline test/extended/openstack/topology.go:27]: nit: leftover

**Comment by MaysaMacedo** [comment]
/lgtm

as Ramon already checked and last commit was just a nit

File: test/extended/openstack/client/client.go [hunk 4]
```diff
@@ -20,7 +21,9 @@ func getProviderClient(ctx context.Context) (*gophercloud.ProviderClient, gopher
 	defer mutex.Unlock()
 
 	if providerClient == nil {
-		ao, eo, tlsConfig, err := clouds.Parse()
+		ao, eo, tlsConfig, err := clouds.Parse(clouds.WithCloudName("overcloud"))
+		e2e.Logf("WOW1 %v", ao.TenantName)
```
Comment (MaysaMacedo) [inline test/extended/openstack/client/client.go:25]: unnecessary change?
